diff --git a/v10.js b/v10.js
index bd0ea59307626d5104da9181fcfe78d360f6e7fa..87e6adb4ae4d8416ec2bc22e5a9c6d66370ee373 100644
--- a/v10.js
+++ b/v10.js
@@ -15,10 +15,10 @@ var __exportStar = (this && this.__exportStar) || function(m, exports) {
 };
 Object.defineProperty(exports, "__esModule", { value: true });
 exports.Utils = void 0;
-__exportStar(require("./gateway/v10"), exports);
-__exportStar(require("./globals"), exports);
-__exportStar(require("./payloads/v10/index"), exports);
-__exportStar(require("./rest/v10/index"), exports);
-__exportStar(require("./rpc/v10"), exports);
-exports.Utils = require("./utils/v10");
+__exportStar(require("./gateway/v10.js"), exports);
+__exportStar(require("./globals.js"), exports);
+__exportStar(require("./payloads/v10/index.js"), exports);
+__exportStar(require("./rest/v10/index.js"), exports);
+__exportStar(require("./rpc/v10.js"), exports);
+exports.Utils = require("./utils/v10.js");
 //# sourceMappingURL=v10.js.map
